scalafix-core/src/main/scala/scalafix/rewrite/ExplicitReturnTypes.scala

```diff
@@ -58,14 +85,39 @@ case class ExplicitReturnTypes(mirror: Mirror)
         typ <- defnType(defn)
       } yield ctx.addRight(replace, s": ${typ.treeSyntax}")
     }.to[Seq]
+
+    import ctx.config.explicitReturnTypes._
+
+    def matchesMemberVisibility(mods: Seq[Mod]): Boolean =
+      memberVisibility.contains(visibility(mods))
+
+    def matchesMemberKind(defn: Defn): Boolean =
+      kind(defn).exists(memberKind.contains)
+
+    def matchesSimpleDefinition(rhs: Term): Boolean =
+      rhs.is[Lit] && skipSimpleDefinitions
```

Comment:
I think I prefer to have the flexibility to redefine simple in future releases. We could make the option an enum, `skipDefinitions = literals/simple`, but I think that's overkill for now. Let's keep it `skipSimpleDefinitions`.